src/configuration.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Gitpod. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+// Use these functions instead of `vscode.workspace.getConfiguration` API
```

Comment:
Is it a problem only with default values or at user setttings are not loaded as well yet?